manifests/init.pp

```diff
@@ -163,7 +164,13 @@
     value    => $max_execution_time,
   }
 
-  # We previously had the max execution time in general settings, 
+  php::apache::config { 'php-expose-php':
```

Comment:
It seemed this should live with all the other options that also are structured the same way. The apache server class does not contain any of the php apache config.